package.json

```diff
@@ -171,5 +171,8 @@
     ],
     "volta": {
         "node": "16.17.0"
+    },
+    "dependencies": {
+        "react-colorful": "^5.6.1"
```

Comment:
//Task for Tyler or Julie - Ensure that this doesn't get bundled with rollup process.

Reply:
Confirming that the the package doesn't get rolled up with the bundle! Here's why:  https://github.com/vimeo/iris/blob/main/rollup.config.js#L29-L30